src/steps/campaign-member/campaign-member-field-equals.ts

```diff
@@ -33,6 +33,24 @@ export class CampaignMemberFieldEquals extends BaseStep implements StepInterface
     type: FieldDefinition.Type.ANYSCALAR,
     description: 'Expected field value',
   }];
+  protected expectedRecords: ExpectedRecord[] = [{
+    id: 'campaignMember',
+    type: RecordDefinition.Type.KEYVALUE,
+    fields: [{
+      field: 'Id',
+      type: FieldDefinition.Type.STRING,
+      description: "Campaign Member's SalesForce ID",
+    }, {
+      field: 'CreatedDate',
+      type: FieldDefinition.Type.DATETIME,
+      description: "Lead's Created Date",
+    }, {
+      field: 'LastModifiedDate',
+      type: FieldDefinition.Type.DATETIME,
+      description: "Lead's Last Modified Date",
+    }],
```

Comment:
- [ ] We should also add `CampaignId`, `LeadId`, `ContactId`, and `LeadOrContactId`, since these are all standard to the `CampaignMember` object and they would all be useful as tokens.